sklearn/tree/tests/test_export.py

```diff
@@ -234,3 +244,27 @@ def test_friedman_mse_in_graphviz():
 
     for finding in finditer("\[.*?samples.*?\]", dot_data.getvalue()):
         assert_in("friedman_mse", finding.group())
+
+
+def test_decimals():
+    rng = RandomState(2)
+    X = rng.random_sample((5, 2))
+    y_reg = rng.random_sample((5, ))
+
+    # regression case
+    clf = DecisionTreeRegressor(criterion="friedman_mse", random_state=0,
+                                max_depth=1)
+    clf.fit(X, y_reg)
+    for decimals, nb_len in zip((4, 3), (5, 4)):
+        dot_data = StringIO()
+        export_graphviz(clf, out_file=dot_data, decimals=decimals)
+        # check value
+        for finding in finditer("nvalue = \d+\.\d+", dot_data.getvalue()):
+            assert_equal(len(search("\.\d+", finding.group()).group()), nb_len)
```

Comment:
It could be less_equal. But with this random seed, we should get exactly `decimals + 1`. Making it less_equal can make the test too much permissive